tables/automl/automl_tables_model.py

```diff
@@ -156,12 +167,24 @@ def get_model(project_id, compute_region, model_display_name):
     else:
         deployment_state = "undeployed"
 
+    # get features of top importance
+    feat_list = [
+        (x.feature_importance, x.column_display_name)
```

Comment:
Can we use `feature` or something similar instead of `x` pretty please?

Reply:
will do